index.html

```diff
@@ -0,0 +1,110 @@
+            </div>
+            <div class="nav-bar-mobile">
+                <div class="black-nav">
+                    <ul class="ul-black">
+                        <li class="github"><a href="" class="a"><i class="fa-brands fa-github"></i></a></li>
+                        <li class="facebook"><a href="" class="a"><i class="fa-brands fa-facebook"></i></a></li>
+                        <li class="anything"><a href="" class="a">Spanish</a></li>
+                    </ul>
+                </div>
+                <div class="mobile-logo">
+                    <button type="button" class="hamburger"><i class="fa-solid fa-bars"></i></button>
+                    <img src="./images/josue-logo.png" alt="josue-design" class="logo-img">
+                </div>
+                
+                <ul class="navigation">
+                    <li class="logo"><img src="./images/josue-logo.png" alt="josue-design" class="logo-img"></li>
+                    <li class="menu"><a href="#intro">Home</a></li>
+                    <li class="menu"><a href="#program">Program</a></li>
+                    <li class="menu"><a href="#speakers">Speakers</a></li>
+                    <li class="menu"><a href="#sponsor">Sponsors</a></li>
+                    <li class="about"><a href="./about.html">About</a></li>
+                </ul>
+            </div>
+        </header>
+        <div id="intro" class="intro">
+            <h2 class="hello">"Hello! Sharing world"</h2>
+            <h1 id="title" class="title">INTERNATIONAL BOOK FAIR IN JESUS MARIA 2022 </h1>
+            <p class="description">You are invited to the International Book Fair in Lima (IBF)
+                that will take place on Proceres park in Jesus Maria</p>
+            <h3 class="date">22 July, 2022</h3>
+            <p class="location">Lima, Perú</p>
+        </div>
+    </div>
+    <div id="program" class="main-program">
+        <h2 class="subtitle">Main Program</h2>
+        <div class="linea"></div>
+        <div class="div-list">
+            <div class="item-list">
+                <img src="./images/list1.png" alt="conference" class="images">
+                <h3 class="img-title">Opinion</h3>
+                <p class="img-des">Listen to the Speakers experts of the art of writing</p>
+            </div>
+            <div class="item-list">
+                <img src="./images/list2.png" alt="conversation" class="images">
+                <h3 class="img-title">Conversation</h3>
+                <p class="img-des">Share your ideas and opinions with other people in the place</p>
+            </div>
+            <div class="item-list">
+                <img src="./images/list3.png" alt="buy" class="images">
+                <h3 class="img-title">Store</h3>
+                <p class="img-des">The most variable selection of books</p>
+            </div>
+        </div>
+        <a href="#" class="square">See whole program</a>
+        <div class="space"></div>
+    </div>
```

Comment:
- [ ] Kindly add 3 more programs like the original. Also please follow the [layout](https://github.com/microverseinc/curriculum-html-css/blob/main/capstone/html_capstone.md#:~:text=You%20should%20follow,space%20between%20elements.)
- [ ] See the whole program is not button and not red in the original kindly change it
- [ ] Please don't forget the mobile view layout after add 3 programs.
![image](https://user-images.githubusercontent.com/38635158/176968726-41031763-e12c-4d4b-9d3d-83da66ccb066.png)
___
### ORIGINAL
![image](https://user-images.githubusercontent.com/38635158/176968756-cfba428f-954c-4c83-8d45-7081705a28ae.png)

____
### YOUR DESIGN MOBILE VIEW
![image](https://user-images.githubusercontent.com/38635158/176969014-ff751817-e5d7-4b5e-9ce2-2f58a2daef7d.png)

___
### ORIGINAL MOBILE VIEW
![image](https://user-images.githubusercontent.com/38635158/176969086-6abcdaf2-8b3d-48c1-8dbf-2cd2ba996903.png)